docs/UserGuide.md

```diff
@@ -222,8 +236,9 @@ A: Delete the data folder to allow _**UDT**_ to create a new data file. Your pre
 Q: Manual insertion of students takes time, is there a faster way to do it?  
 A: We are working on a feature to allow importing of **.csv** files into _**UDT**_!
 
-
-
+Q: What if I key in the wrong command?
```

Comment:
Need to add a double space (aka. line break) to force the answer to be in the next line